runtime/oti/j2sever.h

```diff
@@ -24,20 +24,16 @@
 
 /**
  * Constants for supported J2SE versions.
- * note: J2SE_15 needed for shared classes cache introspection but not supported by JVM.
  */
 /*
  * Note: J2SE_LATEST is the highest Java version supported by VM for a JCL level.
  *       This allows JVM operates with latest version when neither classlib.properties
  *       nor release file presents.
  */
-#define J2SE_15   0x1500
-#define J2SE_16   0x1600
-#define J2SE_17   0x1700
-#define J2SE_18   0x1800
-#define J2SE_19   0x1900
-#define J2SE_V10  0x1A00            /* This refers Java 10 */
-#define J2SE_V11  0x1B00            /* This refers Java 11 */
+#define J2SE_18   0x0800
+#define J2SE_19   0x0900
+#define J2SE_V10  0x0A00            /* This refers Java 10 */
```

Comment:
Please add a comment to explain the constant values are used by the shared cache and the values of the constants must match the version of Java.

Reply:
Is this requires many changes, better done in a separate commit.

Reply:
Yeah, that's true, never mind for this PR then.